src/datascience-ui/interactive-common/variableExplorerCellFormatter.css

```diff
@@ -1,17 +1,17 @@
 .react-grid-variable-explorer-cell-variable {
-    color: var(--code-variable-color, var(--vscode-editor-foreground));
+    color: var(--vscode-editor-foreground);
```

Comment:
Just to check, this was a request from design? Previously they wanted the variable colors in here.

Reply:
I was going off of the Figma mockup, which didn't have any colors in the columns. Maybe that wasn't intentional though? Was Jill the designer who originally requested the variable colors?

Reply:
No it was her compatriot that moved off of our stuff? I can't remember her name though.

Reply:
Yeah, I don't really have a personal stake either way. The type coloring was maybe helpful, but a bit garish at times. If the new mockup doesn't have 'em it seem fine to remove.

Reply:
@jilljac

Reply:
Just confirming, this would essentially apply syntax highlighting within the variable explorer columns? If so, it seems like a reasonable case, as long as it doesn't cause any conflicts for accessibility on keyboard focus.

Reply:
@jilljac More the other way around. The current implementation does use code theme highlighting. This removes it, which is how I believe the current mockup looks. This is how it used to look:
![image](https://user-images.githubusercontent.com/812783/119419267-c4fa4700-bcae-11eb-915c-3e289260abbc.png)
Personally I kinda like Joyce's look in her current pics, but I could see some people wanting their theme reflected here?

Reply:
Let's go with the version that matches Joyce's pics, which has less visual noise and looks more cohesive with the overall design of VS Code. Maybe we can investigate making the syntax highlighting an option later.